### ISSUE TITLE
Many functions fail for unknown reasons, error "No parameterless constructor defined."

### ISSUE THREAD

**Issue by cerebrate** [xoofx/kalk#36]
This is what I'm seeing on debian sid under WSL:

```
>>> cos 45
Cannot dynamically create an instance of type 'Kalk.Core.KalkDoubleValue'. Reason: No parameterless constructor defined.
```

I've verified this with the trig functions, _log_, _sqrt_, _exp_, _round_, etc. _abs_ produces a similar message referencing 'Kalk.Core.KalkCompositeValue'.

I'm not sure why this is happening, and my .NET Core 7 installation seems fine otherwise. Any particular information I should send you for this one? Not sure what would be helpful.

**Comment by xoofx** [comment]
Indeed, I assume that you downloaded one of the binaries from the [releases](https://github.com/xoofx/kalk/releases). It seems that .NET 7 is using a more aggressive code trimming.  This should be fixed by commit 0727d28 and available in a new version 0.8.1 soon.

**Comment by Reedbeta** [comment]
FYI, the Kalk homepage still links to version 0.8.0 which is broken due to this problem. Can that be updated?

**Comment by xoofx** [comment]
> FYI, the Kalk homepage still links to version 0.8.0 which is broken due to this problem. Can that be updated?

Oops, indeed, thanks for reporting. This is fixed by commit 2650cb6
